distribution/integration-tests/src/test/java/org/jboss/tools/rsp/itests/RSPCase.java

```diff
@@ -80,6 +94,30 @@ public static void dispose() throws Exception {
 		RSPServerHandler.stopServer();
 		RSPServerHandler.restoreBackupData(true);
 	}
+	
+	protected void startServer(DummyClient client, String id) throws Exception {
+        Map<String, Object> attr = new HashMap<>();
```

Comment:
Is the formatting off in this file, or is it just github not being nice to me?

Reply:
I think both.. :)